### PR TITLE
fix #56 toggle nav-bar on clicking nav-links

### LINKED ISSUE #56: The mobile nav-bar should close when clicking on any nav-link
screen shot 2018-05-16 at 1 14 39 pm

### PR DESCRIPTION
**Pull request by architkshk**
#56

### CODE REVIEW

File: app/pods/components/nav-bar/template.hbs [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <span class="hamburger-parent" onclick={{action 'toggleSidebar'}}>
             <i class="hamburger fas fa-bars"></i>
           </span>
-    <span>
-      {{#link-to 'index' activeClass=''}}
+    <span class="logo-parent">
+      {{#link-to 'index' activeClass='logo-link'}}
```
Comment (abhishek97) [inline app/pods/components/nav-bar/template.hbs]: Why is this required here?

Comment (architkshk) [reply app/pods/components/nav-bar/template.hbs]: I've updated the code and remove these extra classes.
